### PR TITLE
Assignment 4b finished - Finally got the router to work.

### PR DESCRIPTION
**Pull request by t80532292**
Please grade the assignment, I've finally figure out the router part and get it going. This is my first time using the "pull request". Btw, I've used the .gitignore to filter out the npm_modules.

Dennis Yip.

### CODE REVIEW

**Comment by jamdon** [comment]
I forget whose github account this is.

**Comment by t80532292** [comment]
It is mine. Dennis Yip

Date: Sat, 9 May 2015 20:15:51 -0700

File: app.js [hunk 1]
```diff
@@ -1,36 +1,13 @@
 var express = require('express');
-var trucks = require('./trucks');
-var bodyParser = require('body-parser');
+var router = require('./routes/truckRouter');
```
Comment (jamdon) [inline app.js:2]: router was part of assignment for Module_5

**Comment by jamdon** [comment]
Works good, although the router was part of the assignment for Module_5 not Module_4b. :+1:

**Pull request closed without merging**